docs/source/faq.rst

```diff
@@ -714,3 +714,39 @@ Optuna may sometimes suggest parameters evaluated in the past and if you would l
 
     study = optuna.create_study()
     study.optimize(objective, n_trials=100)
+
+.. _remove_for_artifact_store:
+
+How can I delete all the artifacts uploaded to a study?
+-------------------------------------------------------
+
+Optuna supports `artifacts <https://optuna.readthedocs.io/en/stable/reference/artifacts.html>`_ for large data storage during an optimization.
```

Comment:
```suggestion
Optuna supports `artifacts <https://optuna.readthedocs.io/en/stable/reference/artifacts.html>`__ for large data storage during an optimization.
```

Reply:
@nabenabe0928 Similar to the suggestion in  https://github.com/optuna/optuna/pull/5501#discussion_r1651961644, I think we should add a custom anchor to detect broken links. What do you think?